src/test/suite/statusBar.test.ts

```diff
@@ -0,0 +1,32 @@
+import { expect } from "chai";
+import { onStateChangedEmitter } from "../../events/onStateChangedEmitter";
+import { State } from "../../binary/state";
+import { getStatusBarData } from "../../statusBar/statusBar";
+
+suite("Status bar tests", () => {
+  test("Should change status bar data to 'Failed' when state request returns failure and then get back to 'Ok'", () => {
```

Comment:
1. separate to 2 tests: ok change to failed, and vice versa
2. add a test for service level change